install_script.sh.template

```diff
@@ -330,25 +330,46 @@ if [ -n "$DD_FIPS_MODE" ]; then
 fi
 
 declare -a apm_libraries
+declare host_injection_enabled
+declare docker_injection_enabled
 
-host_injection_enabled=
-if [[ "$DD_APM_HOST_INJECTION_ENABLED" == "true" || ( -z "$DD_APM_HOST_INJECTION_ENABLED" && "$DD_APM_INSTRUMENTATION_ENABLED" == "true") ]] ; then
-  if [ "$no_agent" ]; then
-      ERROR_MESSAGE="Host injection is not supported with the DD_NO_AGENT_INSTALL flag. Disable host injection with DD_APM_HOST_INJECTION_ENABLED=false or unset DD_NO_AGENT_INSTALL"
-      ERROR_CODE=$INVALID_PARAMETERS_CODE
-      echo -e "\033[31m$ERROR_MESSAGE\033[0m\n"
-      report_telemetry
-      exit 1;
-  fi
+if [ -n "$DD_APM_INSTRUMENTATION_ENABLED" ]; then
+  read -r -a target_array <<< "$DD_APM_INSTRUMENTATION_ENABLED"
+  for target in "${target_array[@]}"
+  do
+    case $target in
+      all)
+        host_injection_enabled="true"
+        docker_injection_enabled="true"
+        ;;
+      docker)
+        docker_injection_enabled="true"
+        no_agent=true
```

Comment:
Does this mean that when I specify `host,docker`, I will end up with `no_agent=true`? I think that's not what we want, am I right?

Reply:
Good point, I've removed the support for multiple values. "all" will cover it.